src/ethereum_test_forks/base_fork.py

```diff
@@ -63,9 +75,20 @@ def __call__(
         contract_creation: bool = False,
         access_list: List[AccessList] | None = None,
         authorization_list_or_count: Sized | int | None = None,
+        return_cost_deducted_prior_execution: bool = False,
     ) -> int:
         """
         Returns the intrinsic gas cost of a transaction given its properties.
+
+        The `calldata` parameter is the data of the transaction.
+        The `contract_creation` parameter is whether the transaction creates a contract.
+        The `access_list` parameter is the list of access lists for the transaction.
+        The `authorization_list_or_count` parameter is the list of authorizations or the count of
+        authorizations for the transaction.
+        If `return_cost_deducted_prior_execution` parameter if set to False, the returned value
+        is equal to the minimum gas required for the transaction to be valid. If set to True, the
+        returned value is equal to the cost that is deducted from the gas limit before the
+        transaction starts execution.
```

Comment:
Recommend using the mkdocstrings feature for parameters to render docs with consistent formatting.
```suggestion
        Args:
            calldata: The data of the transaction.
            contract_creation: Whether the transaction creates a contract.
            access_list: The list of access lists for the transaction.
            authorization_list_or_count: The list of authorizations or the count of authorizations for the transaction.
            return_cost_deducted_prior_execution: If set to False, the returned value is equal to the minimum gas required for the transaction to be valid. If set to True, the returned value is equal to the cost that is deducted from the gas limit before the transaction starts execution.            

        Returns:
            Gas cost of a transaction
```

Reply:
I don't know what happened here I thought I had applied the changes, but I think I force pushed over them :see_no_evil:

I'll do another commit.